src/modules/contact/components/icon/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import styles from "./style.module.css";
+
+class Icon extends Component {
+  render() {
+    return (
+        <section className="col-md m-auto">
+            <header>
+                <h1 style={{color: 'var(--text-secondary)'}}>CONTACT US</h1>
+            </header>
+            {/* Message Icon */}
+            <svg
+                className={`bi bi-envelope ${styles.messageIcon}`}
+                viewBox="0 0 16 16"
+                >
+                <path
+                    fillRule="evenodd"
+                    d="M14 3H2a1 1 0 00-1 1v8a1 1 0 001 1h12a1 1 0 001-1V4a1 1 0 00-1-1zM2 2a2 2 0 00-2 2v8a2 2 0 002 2h12a2 2 0 002-2V4a2 2 0 00-2-2H2z"
+                    clipRule="evenodd"
+                />
+                <path
+                    fillRule="evenodd"
+                    d="M.071 4.243a.5.5 0 01.686-.172L8 8.417l7.243-4.346a.5.5 0 01.514.858L8 9.583.243 4.93a.5.5 0 01-.172-.686z"
+                    clipRule="evenodd"
+                />
+                <path d="M6.752 8.932l.432-.252-.504-.864-.432.252.504.864zm-6 3.5l6-3.5-.504-.864-6 3.5.504.864zm8.496-3.5l-.432-.252.504-.864.432.252-.504.864zm6 3.5l-6-3.5.504-.864 6 3.5-.504.864z" />
+            </svg>
```

Comment:
Add visually hidden meaning to the icon to be accessible by screen readers, u can use `sr-only` class provided by bootstrap.